### PR TITLE
Add Storybook to the project

### PR DESCRIPTION
**Pull request by ArpitAgrawal001**

### CODE REVIEW

File: whale [hunk 1]
```diff
@@ -1,5 +1,7 @@
 docker run \
     -u "$(id -u):$(id -g)" \
     -v $(pwd):/work \
-    -w /work -it -p 3000:3000 \
+    -w /work -it \
+    -p 3000:3000 \ # React port
```
Comment (mavenik) [inline whale:5]: @ArpitAgrawal001 Was this tested locally?
Retaining these comments wasn't working, right?

**Review by mavenik — COMMENTED** [review]
@ArpitAgrawal001 Please confirm the working status of your `whale` file.

**Comment by ArpitAgrawal001** [comment]
Yes sir locally it wasn't working

**Comment by ArpitAgrawal001** [comment]
Sir should i merge my PR?

**Comment by ArpitAgrawal001** [comment]
> @ArpitAgrawal001 Please confirm the working status of your `whale` file.

Sir it is working now

**Comment by mavenik** [comment]
Yes @ArpitAgrawal001 
Please go ahead and merge this.

**Pull request merged**